index.js

```diff
@@ -525,16 +526,21 @@ class ResembleHelper extends Helper {
       throw new Error('Cannot get element size!');
     }
 
-    const bottom = size.height + location.y;
-    const right = size.width + location.x;
+    const scrollbarCoordinates = await browser.execute('return { offsetX: window.pageXOffset, offsetY: window.pageYOffset }');
+
+    const bottom = size.height + location.y - scrollbarCoordinates.offsetY;
```

Comment:
I would re-order the operands :) I know it sound silly, but you want add the height to the y. It's just for better readability.
This: ` location.y + size.hight` makes better sense to me.